components/freshservice/freshservice.app.mjs

```diff
@@ -1,4 +1,4 @@
-import { axios } from "@pipedream/platform"
+import { axios } from "@pipedream/platform";
```

Comment:
Looks like this file was ESLint'd on merge w/ master - I'm ignoring the version checks regarding this since it's just formatting